src/pages/leaderboard/index.tsx

```diff
@@ -77,6 +77,17 @@ const Component: FC = () => {
         <Warning />
       </div>
 
+      <div className="stats">
+        <div className="item">
+          <span className="label">Total Value of Airdrop Vaults</span>
+          <span className="value">$1.000.000</span>
```

Comment:
this one is hard coded?

Reply:
@Ehsan-saradar that should pull value of all wallets.